src/main/java/com/amihaiemil/versioneye/RtRepositories.java

```diff
@@ -43,6 +43,9 @@
  * @author Sherif Waly (sherifwaly95@gmail.com)
  * @version $Id$
  * @since 1.0.0
+ * @todo #112:30min/DEV Modify repository(repositoryKey) method to return
```

Comment:
@SherifWaly is it so different? If so, then maybe we need another class indeed, as you initially suggested